src/main/java/de/retest/recheck/ignore/PersistentFilter.java

```diff
@@ -0,0 +1,32 @@
+package de.retest.recheck.ignore;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.Delegate;
+
+@ToString
+@AllArgsConstructor
+public class PersistentFilter implements Filter {
+
+	@Getter
+	private final Path path;
+
+	@ToString.Include
```

Comment:
You would need to specify `@ToString(onlyExplicitlyIncluded = true)`  as described [here](https://projectlombok.org/features/ToString)